src/lib/src/viewer/viewer.component.scss

```diff
@@ -12,11 +12,16 @@
   flex-grow: 1;
 }
 
+:host>>>.openseadragon-canvas:focus {
```

Comment:
Use ::ng-deep instead

(https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep)